test/test_mcmc.py

```diff
@@ -84,6 +84,21 @@ def model(labels):
         assert samples['coefs'].dtype == np.float64
 
 
+def test_uniform_normal():
+    true_coef = 0.9
+
+    def model(data):
+        alpha = sample('alpha', dist.Uniform(0, 1))
+        loc = sample('loc', dist.TransformedDistribution(
+            dist.Uniform(0, 1), constraints.AffineTransform(0, alpha)))
+        sample('obs', dist.Normal(loc, 0.1), obs=data)
+
+    data = true_coef + random.normal(random.PRNGKey(0), (1000,))
+    init_params, potential_fn, constrain_fn = initialize_model(random.PRNGKey(2), model, data)
+    samples = mcmc(1000, 1000, init_params, potential_fn=potential_fn, constrain_fn=constrain_fn)
+    assert_allclose(np.mean(samples['loc'], 0), true_coef, atol=0.05)
```

Comment:
This test will fail with a large margin if dynamic support is not supported. Depending on the initial value of `alpha`, initial support of `loc` will be (0, alpha). So if the initial `alpha` < 0.8, we can not make `loc` near the true value: 0.9.